ballerina/client.bal

```diff
@@ -651,18 +652,55 @@ public isolated client class Client {
 
     }
 
-    # Get query job results
+    # Get bulk query job results
+    # 
     # + bulkJobId - Id of the bulk job
-    # + return - string[][] if successful else `error`
-    isolated remote function getQueryResult(string bulkJobId)
+    # + maxRecords - The maximum number of records to retrieve per set of results for the query
+    # + return - The resulting string[][] if successful else `error`
+    isolated remote function getQueryResult(string bulkJobId, int? maxRecords = ())
```

Comment:
We better make this a defaultable parameter instead (default value should be equal to the salesforce default)
```suggestion
    isolated remote function getQueryResult(string bulkJobId, int maxRecords = 50000)
```